### PR TITLE
Homepage new copy

### PR DESCRIPTION
**Pull request by jameshupp**
## Dependencies
 - [x] Merge #661 into the `homepage_new_copy` branch.
 - [x] Merge [cg-style 237](https://github.com/18F/cg-style/pull/237) into `master` in that repo.

## Changes
* New copy for the homepage, streamlined and focused on customer business. We’re reserving all details about how the platform works and the technology it’s based on to internal content.
* Updated contact.html, which is a partial that at some point we probably need to rename. It includes a number of things, but it’s basically the full set of “next steps” a person might take who is interested to some degree or another
* Lower case ‘g’ in the government banner at the top. This is consistent across 18F products. That’s not necessarily a reason to do it in cloud.gov, but if nobody has a reason to oppose it I want to keep this change.

## Review
* @nikzei 
* @vz3 (or anyone else from Navigator — see Notes below)

## Notes
* I’m going to file an issue for the folks from Navigator momentarily. There are a couple design issues that the new content presents for the homepage. *Please do not merge until we incorporate those changes.*

### CODE REVIEW

**Comment by jameshupp** [comment]
@thisisdano @msecret @nikzei 

I added two checkboxes at the beginning of this PR's first comment. This will be ready to merge when those are handled, so if you can check them off when ready that'd be great. I'll follow up on anything outstanding next week.

File: layouts/index.html [hunk 1]
```diff
@@ -138,90 +111,20 @@ <h2 class="billboard-second_text">Focus on developing your applications. We mana
...
-      <p>
-        Because it’s an agile project, cloud.gov is always under development. As we observe its use, we will continue to add new capabilities. Therefore, we do not say it is “complete” and we probably never will.
-      </p>
+      <h1>Are you right for cloud.gov?</h1>
+      <p>cloud.gov was built by a government team for government teams. Here’s who we’re looking
+        for:</p>
+        <ul>
+          <li>You’re in a federal government organization that employs teams — either in-house
+            or on contract — with the power to push applications.</li>
+          <li>You or your teams can can use the IAA/MOU process to pay GSA for cloud.gov.</li>
```
Comment (adborden) [inline layouts/index.html]: typo "can can"

File: layouts/index.html [hunk 2]
```diff
@@ -138,90 +111,20 @@ <h2 class="billboard-second_text">Focus on developing your applications. We mana
...
+          <li>Your applications are FISMA-Moderate or lower, and the engineering approach values
+              clarity (for example, the application listens to a single port and stores
+              configuration in the environment).</li>
```
Comment (adborden) [inline layouts/index.html]: These sound like two completely separate ideas. Shouldn't they be their own line items?

File: layouts/index.html [hunk 3]
```diff
@@ -118,13 +90,14 @@ <h2 class="billboard-second_text">Focus on developing your applications. We mana
...
-          Where cloud.gov goes from here will depend on how your team uses it today. Already, we
-          are exploring additional products like domain microservices and access to third-party
-          tools. And because every team can benefit from improvements driven by other teams,
-          cloud.gov is an always-improving economy of scale for government teams. This is Government
-          as a Service.
+          Because cloud.gov enforces a consistent environment, no other team using the platform
+          can change the environment in a way that hurts your security or compliance. Because
+          every update applies to every application running on the platform, your team benefits
+          from every improvement. Because we update the environment by deploying a whole new
+          system instead of modifying a live system, any foothold bad actors have found is wiped
```
Comment (adborden) [inline layouts/index.html]: How about "any foothold a bad actor _might_ have found"?

File: layouts/partials/header.html [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <!-- start header -->
 <div class="usa-disclaimer">
   <div class="usa-grid">
-    <span class="usa-disclaimer-official">An official website of the United States Government
+    <span class="usa-disclaimer-official">An official website of the United States government
       <img alt="US flag signifying that this is a United States Federal Government website" src="/assets/img/us_flag_small.png">
```
Comment (adborden) [inline layouts/partials/header.html]: Shouldn't Federal Government be lowercase, too, since it's not a title/name?

File: layouts/index.html [hunk 5]
```diff
@@ -138,90 +85,17 @@ <h2 class="billboard-second_text">Focus on developing your applications. We mana
...
-      </p>
-      <p>
-        Because it’s an agile project, cloud.gov is always under development. As we observe its use, we will continue to add new capabilities. Therefore, we do not say it is “complete” and we probably never will.
-      </p>
+      <h1 class="section-title">Are you right for cloud.gov?</h1>
+      <p class="section-dek">cloud.gov was built by a government team for government teams. Here’s who we’re looking
+        for:</p>
+        <ul class="list-lined">
+          <li>You’re in a federal government organization that employs teams — either in-house or on contract — with the power to push applications.</li>
+          <li>You or your teams can can use the IAA/MOU process to pay GSA for cloud.gov.</li>
```
Comment (adborden) [inline layouts/index.html]: @thisisdano would you resolve the merge conflicts and fix this typo ^^

File: layouts/index.html (hunk 3 above)
Comment (thisisdano) [reply layouts/index.html]: Ha, that sounds reasonable.

File: layouts/index.html (hunk 2 above)
Comment (thisisdano) [reply layouts/index.html]: Seems reasonable.

File: layouts/partials/header.html (hunk 4 above)
Comment (thisisdano) [reply layouts/partials/header.html]: in the alt text? seems like a good call.

**Comment by thisisdano** [comment]
@adborden I think this is ready to go now. Unfortunately, I added a utility class to the homepage billboard H1, as something must've gotten screwed up in the regular css classes, but I wanted it to look right right now, and not wait for a new cg-style release. I can mark this to fix later.

**Review by adborden — APPROVED** [review]

**Pull request merged**